python/python/tests/test_async_client.py

```diff
@@ -4453,6 +4453,64 @@ async def test_object_refcount(self, redis_client: TRedisClient):
         refcount = await redis_client.object_refcount(string_key)
         assert refcount is not None and refcount >= 0
 
+    @pytest.mark.parametrize("cluster_mode", [False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_function_load_standalone(self, redis_client: RedisClient):
+        # TODO: Test function with FCALL
+        # TODO: Test with FUNCTION LIST
+        min_version = "7.0.0"
+        if await check_if_server_version_lt(redis_client, min_version):
+            return pytest.mark.skip(reason=f"Redis version required >= {min_version}")
+
+        lib_name = f"mylib1C{get_random_string(5)}"
+        code = f"#!lua name={lib_name}\n redis.register_function('myfunc1c{get_random_string(5)}', function(keys, args) return args[1] end)"
```

Comment:
Consider creating a function which generates lib code for future use: https://github.com/aws/glide-for-redis/blob/7f4814c756c74b94be0ca037f391400fe3f75386/java/integTest/src/test/java/glide/TestUtilities.java#L202-L203
I rewrote the tests for function commands a lot, please refer to code on main

Reply:
i will